Lib/test/test_zlib.py

```diff
@@ -831,6 +831,12 @@ def test_wbits(self):
         dco = zlib.decompressobj(32 + 15)
         self.assertEqual(dco.decompress(gzip), HAMLET_SCENE)
 
+        for wbits in (-15, 15, 31):
+            self.assertEqual(
+                zlib.decompress(
+                    zlib.compress(HAMLET_SCENE, wbits=wbits), wbits=wbits
+                ), HAMLET_SCENE)
+
```

Comment:
Please use [subTest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest) so that in case of failure we know which value caused the failure.

```suggestion
            with self.subTest(wbits=wbits):
                expected = HAMLET_SCENE
                actual = zlib.decompress(
                    zlib.compress(HAMLET_SCENE, wbits=wbits), wbits=wbits
                )
                self.assertEqual(expected, actual)
```